Solutions/Vectra AI Detect/Analytic Rules/VectraDetect-Account-Detections.yaml

```diff
@@ -71,9 +71,9 @@ entityMappings:
         columnName: name
       - identifier: UPNSuffix
         columnName: upn_suffix
-alertDetails:
-  alertDisplayNameFormat: Vectra AI - {{Activity}} Detected
-  alertDescriptionFormat: >-
+alertDetailsOverride:
+  alertDisplayNameFormat: "Vectra AI - {{Activity}} Detected"
```

Comment:
Additional quotes are not required in alertDisplayNameFormat or alertDescriptionFormat.
Applicable to all the queries.

Reply:
good to know @devikamehra

Reply:
Can we remove these?